nekoyume/Assets/_Scripts/UI/Shop/ShopBuy.cs

```diff
@@ -144,15 +125,14 @@ private async void AsyncShow(bool ignoreShowAnimation = false)
             }
         }
 
-
         private void Reset()
         {
             ShowNPC();
         }
 
         public void Open()
         {
-            ReactiveShopState.Update();
+            // ReactiveShopState.UpdateBuyDigests();
```

Comment:
is it intended?

Reply:
@hskim881028 ⬆️

Reply:
This is an unnecessary comment. I will remove